src/database/DataTypes/StocktakeBatch.js

```diff
@@ -296,6 +317,8 @@ export class StocktakeBatch extends Realm.Object {
       // (i.e. always treat as positive).
       const snapshotDifference = Math.abs(this.difference);
       transactionBatch.setTotalQuantity(database, snapshotDifference);
+      transactionBatch.doses = this.doses;
+
```

Comment:
This is for making an inventory adjustment for a stocktake batch. Make sure the doses is correctly set on the transaction batch